Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -245,5 +240,11 @@ extension CustomerIOImplementation {
                                     blockingGroups: [
                                         .identifiedProfile(identifier: currentlyIdentifiedProfileIdentifier)
                                     ])
+
+        if type == .screen {
```

Comment:
works, but not sure I love it being buried inside `trackEvent` - my gut says it should be duplicated in both of `screen` and `automaticScreenView` instead - doesn't seem likely to expand beyond this size, and then it's colocated with the actual screen call instead of a side-effect inside the trackEvent

... that said, this is a style choice, so if you think this reads / works better in a Swifty world, then 👍 either way